.github/workflows/post-promote-tasks.yml

```diff
@@ -51,6 +53,28 @@ jobs:
                 "tag": "${{ matrix.npm-tag }}",
               }
             }
+
+      - name: Create event on release calendar
+        # run this step even if previous step fails
+        if: ${{ always() }} && ${{ matrix.calendar-channel }}
```

Comment:
I think it would make sense to split this into a separate `jobs` sub-section so it won't depend on the items at npm publishing at all. You could also reorganize the output of the setup job so that it outputs a separate variable for each type of task instead of making it a single JSON object

Reply:
Good point, updated.